src/textual/_node_list.py

```diff
@@ -46,10 +47,10 @@ def __rich_repr__(self) -> rich.repr.Result:
     def __len__(self) -> int:
         return len(self._nodes)
 
-    def __contains__(self, widget: Widget) -> bool:
+    def __contains__(self, widget: object) -> bool:
```

Comment:
otherwise this breaks "LSP" (Laskov Subs. Principle), as `Sequence` defines this to be `object`.